bump-packages/action.yml

```diff
@@ -9,11 +9,11 @@ inputs:
     description: GitHub token (not the default one)
     required: true
   formulae:
-    description: Formula names, one per line
+    description: Formula names, space-separated
     required: false
```

Comment:
Is this a breaking change or was the description wrong before?

Reply:
I believe it was wrong (or rather a misunderstanding of how YAML `>` syntax works).

If it really is one per line then this probably wouldn't work:

https://github.com/Homebrew/actions/blob/250a86a75e69d18d941481f9c02b684363233daf/bump-packages/action.yml#L28

Reply:
Cool, make sense!